deployment_scripts/install-infection-monkey-service.sh

```diff
@@ -0,0 +1,138 @@
+#!/bin/bash
+
+set -e
+
+SCRIPT_DIR="$(realpath $(dirname $BASH_SOURCE[0]))"
+SYSTEMD_UNIT_FILENAME="monkey-appimage.service"
+SYSTEMD_DIR="/lib/systemd/system"
+MONKEY_BIN="/opt/infection-monkey/bin"
+APPIMAGE_NAME="InfectionMonkey.appimage"
+
+echo_help() {
+  echo "usage: install-infection-monkey-service.sh [--user <NAME> --appimage <PATH>] [--help] [--uninstall]"
+  echo ""
+  echo "Installs Infection Monkey AppImage and systemd unit to run on boot"
+  echo "--user                          User to run the AppImage as"
+  echo "--appimage                      Path to the AppImage"
+  echo "--uninstall                     Uninstall Infection Monkey AppImage systemd service"
+}
+
+service_install() {
+  cat > "${SCRIPT_DIR}/${SYSTEMD_UNIT_FILENAME}" << EOF
+[Unit]
+Description=Infection Monkey AppImage Runner
+After=network.target
+
+[Service]
+User=$1
+Type=simple
+ExecStart="${MONKEY_BIN}/${APPIMAGE_NAME}"
+
+[Install]
+WantedBy=multi-user.target
+EOF
+
+  sudo mv "${SCRIPT_DIR}/${SYSTEMD_UNIT_FILENAME}" "${SYSTEMD_DIR}/${SYSTEMD_UNIT_FILENAME}"
+
+  # Enable on boot
+  sudo systemctl enable "${SYSTEMD_UNIT_FILENAME}" &>/dev/null
+  sudo systemctl daemon-reload
```

Comment:
Is running `daemon-reload` necessary? We don't need to recreate the entire dependency tree. Running `enable` should cover it.

![image](https://user-images.githubusercontent.com/19957806/162023290-d8506c4c-81f7-48e1-8e7d-a98b66d4b380.png)
https://man7.org/linux/man-pages/man1/systemctl.1.html